### ISSUE TITLE
Config WriteTimeout property is a circular reference

### ISSUE THREAD

**Issue by eiladin** [harness/ff-dotnet-server-sdk#45]
https://github.com/harness/ff-dotnet-server-sdk/blob/main/client/api/Config.cs#L58

```
public int WriteTimeout { get => WriteTimeout;  }
```

should be 

```
public int WriteTimeout { get => writeTimeout;  }
```

**Comment by sharprs83** [comment]
I noticed that too when I was looking at connection timeouts.

**Comment by sharprs83** [comment]
Looks like this was [fixed in this commit](https://github.com/harness/ff-dotnet-server-sdk/commit/dc7c34736ce464caabd7fbf3aed7aad6749f6749).

**Comment by andybharness** [comment]
Fixed in 1.1.8
